tests/helpers/toLowerCase.test.js

```diff
@@ -1,11 +0,0 @@
-var helpers = require('../../js/main.js');
```

Comment:
Yeah sorry I removed the helpers folder. All of the helpers are in tests/test.js now. I hadn't realised you'd written one. I didn't write the tests file properly until today